index.html

```diff
@@ -1,52 +1,68 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
+<!DOCTYPE html>
+<html lang="en">
 <head>
-<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
-<title>Homepage</title>
-<link rel="stylesheet" type="text/css" href="tryonce.css" /> 
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Homepage</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous"> 
+     <link rel="stylesheet" type="text/css" href="tryonce.css" />
 </head>
 
 <body>
-<h1> Jobeet </h1>
-<a href="post_job.html">Post a job</a>
-<br />
-<a href="find_job.html">Find a job</a>
-<br />
-<a href="user.html">User</a>
-<br />
-<h3> New jobs </h4>
-</table>
-<table class="tablelist">
-    <tr> 
-        <th>Name</th>
-        <th>Posted</th>
-        <th>Action</th> 
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
```

Comment:
Em nên để thêm 1 <div class="container"> bao trùm toàn bộ các thẻ con của <nav> nhé. Kiểu như này: http://prntscr.com/ik7opa